src/Core/Commander/Interfaces/ApiInterface/ApiGlobe.js

```diff
@@ -230,6 +271,7 @@ ApiGlobe.prototype.createSceneGlobe = function(coordCarto, viewerDiv) {
     var coordinate = new GeoCoordinate().copy(coordCarto,UNIT.DEGREE);
 
     this.scene = Scene(coordinate, ellipsoid, viewerDiv, debugMode, gLDebug);
+    this.scene.parent = this;
```

Comment:
Finally, I placed the function inside the Scene. I did not used the event because it was not necessary as I could access to the Scene from the Index. Tell me if that is fine for you or if you want me to change it.